lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -309,3 +311,28 @@ def use_security_token_auth(func):
 #         abort(401)
 #     request.user_token_data = id_info
 #     return
+
+def verify_githubauth(token_str):
```

Comment:
Whole block will not be used. Like @sandeep-ps mentioned, lib/auth-middleware don't need be changed at all. Sorry for the confusion.